### ISSUE TITLE
Set-up email password reset functionality

### ISSUE THREAD

**Issue by dchiller** [DDMAL/VIM#94]
Cantus Database offers a good approach for this

**Comment by Monalisa98927** [comment]
I cannot receive the password reset email from CantusDB. How do I set the following constants for VIM?

Corresponds to line 168-175 here in CantusDB https://github.com/DDMAL/CantusDB/blob/516fa0fb9f11f5e3524a2f93f9439f51cba38cad/django/cantusdb_project/cantusdb/settings.py#L168

**Comment by Monalisa98927** [comment]
> Do you have an account on CantusDB? If not, I think you would not receive an email...

No, I don't have an account. If that's why I didn't receive the verification email, that's fine.

> We will have to configure this in AWS -- I can do this today.

Great! Thank you.

**Comment by Monalisa98927** [comment]
> > No, I don't have an account. If that's why I didn't receive the verification email, that's fine.
> 
> Ah, ok ... that makes sense.
> 
> I will get to the config today....sorry I lost track of it yesterday.

@dchiller Hi Dylan! Is the AWS configuration ready?

**Comment by dchiller** [comment]
I will tag in @softcat477 here. Do you know how to set up a new email address in AWS? Also, maybe we don't want to do this yet because I think only simssa.ca exists but we would want this to be linkedmusic.ca for vim. And so it doesn't really make sense to do this twice.

**Comment by fujinaga** [comment]
This process should be documented so that anyone can set this up.

**Comment by softcat477** [comment]
> I will tag in @softcat477 here. Do you know how to set up a new email address in AWS? 

The email service is on Amazon Simple Email Service (`us-west-2`). There's no documentation about how to set up a new email address. I can look into this on Thursday

> Also, maybe we don't want to do this yet because I think only simssa.ca exists but we would want this to be linkedmusic.ca for vim. And so it doesn't really make sense to do this twice.

I agree. We should worry about email service after switching to `linkedmusic.ca`.

**Comment by dchiller** [comment]
Ok, let's put this on hold for now then.
